config/rbac/role.yaml

```diff
@@ -18,7 +18,7 @@ rules:
   resources:
   - apiexports/content
   verbs:
-  - update
```

Comment:
The `kcp/kustomization.yaml` file uses `kcp-rbac` dir, not `rbac` dir, so I don't think this change is really needed, but doesn't hurt I guess

Reply:
Thanks, I agree. It is added when I run `made manifests`.

Reply:
Ah, makes sense!